<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="codingQuestion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>620</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="codeBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>620</width>
     <height>385</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="codeSubmit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>450</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>제출</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
